src/runtime/server/middleware/session/storage.ts

```diff
@@ -1,8 +1,8 @@
-import { createStorage, prefixStorage, StorageValue } from 'unstorage'
+import { prefixStorage, StorageValue } from 'unstorage'
+import { sessionStorage } from './sessionStorage'
 import { useRuntimeConfig } from '#imports'
-
-const storage = prefixStorage(createStorage(useRuntimeConfig().session.session.storageOptions), useRuntimeConfig().session.session.storePrefix)
-
+const sessionConfig = useRuntimeConfig().session.session
+const storage = prefixStorage(sessionStorage, sessionConfig.storePrefix)
```

Comment:
Do you want to move this part into `sessionStorage` file as well (since there you already have `sessionConfig`) and directly import prefixed storage or is there some reason to prefix the storage here?

Reply:
no reason to prefix here, I was just being disorganized :)
refactored this